src/app.js

```diff
@@ -1,21 +1,30 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+
+const [ file, folder ] = [...process.argv.slice(3)];
+
+if (folder[folder.length - 1] !== '/') {
+  throw new Error('The name of destination directory should be ended with /');
 }
 
-module.exports = sum;
+fs.readFile(file, (err, content) => {
+  if (err) {
```

Comment:
The names of `err` `err1` `err2` look magical and don't explain in the code what kind of errors these are. Try to use constructive variable names: `readFileError` etc